.ci/docker/test/camunda-docker-labels.golden.json

```diff
@@ -0,0 +1,21 @@
+{
+  "io.k8s.description": "Workflow engine for microservice orchestration",
```

Comment:
❓ I guess these labels are still need to be updated?

Reply:
right, it is being discussed here https://github.com/camunda/camunda/pull/19495#issuecomment-2188429756
if you have any suggestions let me know :)
otherwise I will reach out to [#ask-devrel](https://camunda.slack.com/archives/CSYM5MBJ5)

Reply:
updated some of the labels here https://github.com/camunda/camunda/pull/19623/commits/514a4c839465949d09ddf76ff96c810c262fc3f3
after this discussion https://camunda.slack.com/archives/C026U8GBNSW/p1719493763579659